### PR TITLE
Increase viewport defaults

### PR DESCRIPTION
**Pull request by ivanprado**

### CODE REVIEW

File: splash/defaults.py [hunk 1]
```diff
@@ -13,9 +13,9 @@
 VIEWPORT_SIZE = '1024x768'
 
 # Window size limitations.
-VIEWPORT_MAX_WIDTH = 20000
-VIEWPORT_MAX_HEIGTH = 20000
-VIEWPORT_MAX_AREA = 4000*4000
+VIEWPORT_MAX_WIDTH = 40000
+VIEWPORT_MAX_HEIGTH = 40000
+VIEWPORT_MAX_AREA = 7200*7200  # ~52 Millions of pixels
```
Comment (kmike) [inline splash/defaults.py]: It could be more clear if this is written as 1280 * 40000 - "support max height @ 1280 width"

Comment (ivanprado) [reply splash/defaults.py]: Much better! I didn't notice where you had you gotten the magic number of 52M :-)

File: splash/defaults.py [hunk 2]
```diff
@@ -13,9 +13,9 @@
 VIEWPORT_SIZE = '1024x768'
 
 # Window size limitations.
-VIEWPORT_MAX_WIDTH = 20000
-VIEWPORT_MAX_HEIGTH = 20000
-VIEWPORT_MAX_AREA = 4000*4000
+VIEWPORT_MAX_WIDTH = 40000
+VIEWPORT_MAX_HEIGTH = 40000
+VIEWPORT_MAX_AREA = 1280*VIEWPORT_MAX_HEIGTH  # support max height @ 1280 width
```
Comment (kmike) [inline splash/defaults.py]: a nitpick: could you please add spaces around `*`, as per pep8?

Comment (ivanprado) [reply splash/defaults.py]: Sure! I didn't put them because I thought it was the standard for the project.

Comment (ivanprado) [reply splash/defaults.py]: Done

**Comment by kmike** [comment]
Thanks @ivanprado!

**Pull request merged**